app/layout.tsx

```
@@ -0,0 +1,131 @@
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en" className={`${inter.variable} ${playfair.variable}`}>
+      <head>
+        <link rel="icon" href="/favicon.ico" />
+        <link rel="apple-touch-icon" href="/lovable-uploads/381ae573-3196-4996-adc1-acbc78af6037.png" />
+        <meta name="theme-color" content="#1e3a5f" />
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)